packages/core/src/clone/CloneManager.ts

```
@@ -103,98 +103,90 @@ export class CloneManager {
+      case Int32Array:
+      case Float32Array:
+      case Float64Array:
+        let targetPropertyT = <TypedArray>target[k];
+        if (targetPropertyT == null || targetPropertyT.length !== (<TypedArray>sourceProperty).length) {
+          target[k] = (<TypedArray>sourceProperty).slice();
+        } else {
+          targetPropertyT.set(<TypedArray>sourceProperty);
+        }
+        break;
+      case Array:
+        let targetPropertyA = <Array<any>>target[k];
```

Biome (2.4.7)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)